samples/boards/stm32/backup_sram/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  description: Usage of backup SRAM on STM32
+  name: stm32_backup_sram
+tests:
+  sample.boards.stm32.backup_sram:
+    depends_on: backup_sram
+    build_only: true
```

Comment:
Would be nice to make it runnable in CI, but this is next step.

Reply:
I guess we need hardware on CI for that?

Reply:
We have a bench and daily tests are run.